pkg/scheduler/framework/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,173 @@
+	*out = *in
+	if in.Namespaces != nil {
+		in, out := &in.Namespaces, &out.Namespaces
+		*out = make(sets.Set[string], len(*in))
+		for key, val := range *in {
+			(*out)[key] = val
+		}
+	}
+	if in.Selector != nil {
+		out.Selector = in.Selector.DeepCopySelector()
+	}
+	if in.NamespaceSelector != nil {
+		out.NamespaceSelector = in.NamespaceSelector.DeepCopySelector()
+	}
+	return
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new AffinityTerm.
+func (in *AffinityTerm) DeepCopy() *AffinityTerm {
+	if in == nil {
+		return nil
+	}
+	out := new(AffinityTerm)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *ClusterEvent) DeepCopyInto(out *ClusterEvent) {
+	*out = *in
+	return
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new ClusterEvent.
+func (in *ClusterEvent) DeepCopy() *ClusterEvent {
+	if in == nil {
+		return nil
+	}
+	out := new(ClusterEvent)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *PodInfo) DeepCopyInto(out *PodInfo) {
+	*out = *in
+	if in.Pod != nil {
+		in, out := &in.Pod, &out.Pod
+		*out = new(v1.Pod)
+		(*in).DeepCopyInto(*out)
+	}
+	if in.RequiredAffinityTerms != nil {
+		in, out := &in.RequiredAffinityTerms, &out.RequiredAffinityTerms
+		*out = make([]AffinityTerm, len(*in))
+		for i := range *in {
+			(*in)[i].DeepCopyInto(&(*out)[i])
+		}
+	}
+	if in.RequiredAntiAffinityTerms != nil {
+		in, out := &in.RequiredAntiAffinityTerms, &out.RequiredAntiAffinityTerms
+		*out = make([]AffinityTerm, len(*in))
+		for i := range *in {
+			(*in)[i].DeepCopyInto(&(*out)[i])
+		}
+	}
+	if in.PreferredAffinityTerms != nil {
+		in, out := &in.PreferredAffinityTerms, &out.PreferredAffinityTerms
+		*out = make([]WeightedAffinityTerm, len(*in))
+		for i := range *in {
+			(*in)[i].DeepCopyInto(&(*out)[i])
+		}
+	}
+	if in.PreferredAntiAffinityTerms != nil {
+		in, out := &in.PreferredAntiAffinityTerms, &out.PreferredAntiAffinityTerms
+		*out = make([]WeightedAffinityTerm, len(*in))
+		for i := range *in {
+			(*in)[i].DeepCopyInto(&(*out)[i])
+		}
+	}
+	if in.cachedResource != nil {
+		in, out := &in.cachedResource, &out.cachedResource
+		*out = new(podResource)
+		(*in).DeepCopyInto(*out)
+	}
+	return
+}
+
+// DeepCopy is an autogenerated deepcopy function, copying the receiver, creating a new PodInfo.
+func (in *PodInfo) DeepCopy() *PodInfo {
+	if in == nil {
+		return nil
+	}
+	out := new(PodInfo)
+	in.DeepCopyInto(out)
+	return out
+}
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *QueuedPodInfo) DeepCopyInto(out *QueuedPodInfo) {
+	clone := in.DeepCopy()
+	*out = *clone
+	return
+}
```

Comment:
![image](https://github.com/user-attachments/assets/47eb9834-9596-42e7-b35c-c316d8fda038)

the deepcopy-gen tool is not working for this case. cc @sanposhiho 

https://github.com/kubernetes/kubernetes/blob/a1a4bc8fec8d5e1e682d85638508dc1381345508/pkg/scheduler/framework/types.go#L372-L373